be/src/Infrastructure/Catalog/Foundation.Catalog/Infrastructure/Products/Entities/Product.cs

```diff
@@ -31,6 +31,8 @@ public class Product : Entity
 
         public string Ean { get; set; }
 
+        public int DaysToFulfilment { get; set; }
```

Comment:
Hi @PiotrZaranek, it's fulfillment with a double 'l' inside. Let's rename it to FulfillmentTime where instead of days the amount of seconds is saved to the database and converted to days on the frontend, please